testcases/OpalUtils.py

```diff
@@ -193,6 +193,7 @@ def runTest(self):
             self.skipTest("Unknown CPU type %s" % self.cpu)
 
         self.utils_init()
+        return
```

Comment:
Can you remove this return from here. I just added this to verify quickly, but it is not needed.